src/PaperG/FirehoundBlob/Facebook/FacebookCreative.php

```diff
@@ -11,12 +10,12 @@ class FacebookCreative implements BlobInterface
 {
     use Utility;
 
-    const TYPE = 'type';
-    const OBJECTS = 'objects';
-    const VERSION = 'version';
-
-    const LINK_TYPE = 'link';
-    const CAROUSEL_TYPE = 'carousel';
+    const TYPE              = 'type';
+    const CHILD_ATTACHMENTS = 'child_attachments';
+    const PRIMARY           = 'primary';
+    const VERSION           = 'version';
```

Comment:
Any need to say version 2, or are we still in the phase where we can pretend it was always like this? :D